interop/jvm/src/main/scala/scalaz/zio/interop/catz.scala

```diff
@@ -97,3 +128,16 @@ trait CatsBifunctor extends Bifunctor[IO] {
   override def bimap[A, B, C, D](fab: IO[A, B])(f: A => C, g: B => D): IO[C, D] =
     fab.bimap(f, g)
 }
+
+import scalaz.zio._
+
+object App extends App {
```

Comment:
Oops, do we really want this in here? Seems like it would be better in `tut` docs.